renderer/src/components/EditorPage/Reference/TA/TaNavigation.js

```diff
@@ -27,41 +27,41 @@ export default function TaNavigation({ languageId, referenceResources }) {
 
   const {
     state: {
-        owner,
-        // taNavigationPath,
+      owner,
+      // taNavigationPath,
     },
     actions: {
-        setTaNavigationPath,
-      },
+      setTaNavigationPath,
+    },
   } = useContext(ReferenceContext);
 
-    useEffect(() => {
-      if (referenceResources && referenceResources?.offlineResource?.offline) {
-        // offline
-        const taArrayOffline = [];
-        const { offlineResource } = referenceResources;
-        // console.log('offline data : ', { taList, offlineResource });
-        localForage.getItem('userProfile').then(async (user) => {
-          logger.debug('TaNavigation.js', 'reading offline helps ', offlineResource.data?.projectDir);
-          const fs = window.require('fs');
-          const path = require('path');
-          const newpath = localStorage.getItem('userPath');
-          const currentUser = user?.username;
-          const folder = path.join(newpath, packageInfo.name, 'users', `${currentUser}`, 'resources');
-          const projectName = `${offlineResource?.data?.value?.meta?.name}_${offlineResource?.data?.value?.meta?.owner}_${offlineResource?.data?.value?.meta?.release?.tag_name}`;
-          // multiple books or options
-          if (offlineResource?.data?.value?.books.length > 0) {
-            setoptions(offlineResource?.data?.value?.books);
-          }
-          // if (fs.existsSync(path.join(folder, projectName, 'translate'))) {
-          if (selectedOption && fs.existsSync(path.join(folder, projectName, selectedOption))) {
-            fs.readdir(path.join(folder, projectName, selectedOption), async (err, folderNames) => {
-              if (err) {
-                // console.log(`Unable to scan directory: ${ err}`);
-                logger.debug('TaNavigation.js', `Unable to scan directory: ${ err}`);
-              }
-              let foldersCount = 0;
-              if (folderNames.length > 0) {
+useEffect(() => {
+    if (referenceResources && referenceResources?.offlineResource?.offline) {
+      // offline
+      const taArrayOffline = [];
+      const { offlineResource } = referenceResources;
+      // console.log('offline data : ', { taList, offlineResource });
+      localForage.getItem('userProfile').then(async (user) => {
+        logger.debug('TaNavigation.js', 'reading offline helps ', offlineResource.data?.projectDir);
+        const fs = window.require('fs');
+        const path = require('path');
+        const newpath = localStorage.getItem('userPath');
+        const currentUser = user?.username;
+        const folder = path.join(newpath, 'autographa', 'users', `${currentUser}`, 'resources');
```

Comment:
@samueljd it should be packageInfo.name instead of 'autographa', we are providing dynamic value

Reply:
I've fixed it.